packages/babel-core/src/tools/build-external-helpers.js

```diff
@@ -61,44 +59,44 @@ function buildGlobal(namespace, builder) {
   return tree;
 }
 
+const normalizeHelperId = helperId => helperId.replace(/^_+/, "");
```

Comment:
Not sure how I feel about making assumptions like this. If we need to whitelist an explicit name, we should be passing that along, rather than assuming it will match up with the exported function name. Also as mentioned above, this whitelist seems like something we should kill if we can anyway.

Reply:
> Not sure how I feel about making assumptions like this. If we need to whitelist an explicit name, we should be passing that along, rather than assuming it will match up with the exported function name.

Didnt like this either, some bigger changes to `builder` would have to be done though, and I've settled for now on a simpler change - as more parts of the build external helpers are based on some assumptions. `module` complicated things a little bit here, as its the only output type which doesn't need a namespace (as it is one). And got kinda (by me :<) glued into existing builders.

> Also as mentioned above, this whitelist seems like something we should kill if we can anyway.

👍 for killing this whitelist, raised that up today on slack - however I would prefer merging this quickly (if rest of the PR is ok), as this is broken at the moment in beta. I can provide a followup PR later.